### PR TITLE
3.0 fix docs links

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

**Comment by benbc** [comment]
@jjaderberg The first three commits are unchanged from your work. Please could you look at the my commit ("Update README links to point to the new manuals"), please?

File: packaging/standalone/permalinks.properties [hunk 1]
```diff
@@ -1,20 +1,14 @@
 neo4j-home.url=http://neo4j.com/
 neo4j-home.url.title=Neo4j Home
-neo4j-manual.url=http://neo4j.com/docs/#{project.version}/
+neo4j-manual.url=http://neo4j.com/docs/
 neo4j-manual.title=The Neo4j Manual
```
Comment (jjaderberg) [inline packaging/standalone/permalinks.properties]: The link no longer leads to The Manual, maybe change the title to "Neo4j Documentation" or similar.

File: packaging/standalone/permalinks.properties [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
 downloads.url=http://neo4j.com/download/
 downloads.url.title=Neo4j Downloads
-getting-started.url=http://neo4j.com/docs/#{project.version}/introduction.html
+getting-started.url=http://neo4j.com/docs/developer-manual/current/#introduction
 getting-started.url.title=Getting Started
-getting-started-server.url=http://neo4j.com/docs/#{project.version}/deployment.html
-getting-started-ha.url=http://neo4j.com/docs/#{project.version}/ha-setup-tutorial.html
+getting-started-server.url=http://neo4j.com/docs/operations-manual/current/#deployment
+getting-started-ha.url=http://neo4j.com/docs/operations-manual/current/#tutorials
 getting-started-server.url.title=Getting Started Java With Neo4j Server
```
Comment (jjaderberg) [inline packaging/standalone/permalinks.properties]: This "title" is messed up, thankfully it is not used anywhere. Could you remove it so it doesn't get used (or fix it)?

**Comment by jjaderberg** [comment]
@benbc 👍

**Comment by benbc** [comment]
@jjaderberg Made the changes you suggested.

**Pull request merged**